vizro-core/docs/pages/user-guides/card-button.md

```diff
@@ -347,9 +357,9 @@ Use the following pre-defined URL hashes in your image path to apply Vizro's def
                 components=[
                     vm.Card(
                         text="""
-                        ### My card with image!
+                        ### My card with floating image!
 
-                        ![](assets/images/continents/africa.svg#my-image#floating-right)
+                        ![](assets/images/continents/europe.svg#my-image#floating-center)
```

Comment:
Can we actually keep the floating-right example? It's what people more commonly would like to do. 

Also the screenshot for that doesn't seem right?

Reply:
The thing was, the example code was for Africa floating right but the screen grab shown was europe floating left! I compromised and put Europe in the middle, so I changed the text and the graphic 😆  I'll revert to do as you mentioned and keep it floating right.

Reply:
https://github.com/mckinsey/vizro/pull/569/commits/e63e383b781c7388edc7053777e0430324b7eb6e